CHANGES.md

```diff
@@ -7,6 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.3.0] - 2021-12-10
+
+### Added
+
+- A precompile which exposes NEAR's random number generator was added by [@mfornet] as requested by 
+[@birchmd]. ([#368] [#297])
+- London hard fork support was added by [@birchmd]. ([#244])
+
+### Changes
+
+- The gas limit for `deposit` and `ft_on_transfer` were changed as they were not attaching enough
+gas, as changed by [@mrLSD]. ([#389])
+
+### Fixes
+
+- There was an issue with the original storage not actually being stored. Unfortunately, previous 
+transactions can't be updated with this change. This has been fixed by [@birchmd]. ([#390])
+- Call arguments were intended to have a value attached to them to make it equivalent to an ETH
+call. This was fixed in a backwards compatible manner by [@andrcmdr], as reported by [@birchmd].
+([#351] [#309])
+- A refund now happens in the case of an error on the `ExitToNear` precompile by [@birchmd]. 
```

Comment:
This was merged behind a feature flag, so it's not actually enabled. Still waiting on @sept-en / @mfornet to do the necessary migration before we can enable this.

Reply:
Will update after the call.

Reply:
Fixed